assets/scss/common/_content-four-column.scss

```diff
@@ -66,7 +66,7 @@
         @include medium-and-up {
           font-size: rem(18);
           font-weight: 500;
-          color: $black;
+          color: $dark-blue;
```

Comment:
Can you move the color on the `h4` block above. It seems that on mobile it has the wrong blue, so let's fix it for all devices.